pkg/services/ngalert/models.go

```diff
@@ -90,3 +90,45 @@ type listAlertDefinitionsCommand struct {
 
 	Result []*AlertDefinition
 }
+
+// AlertInstance represent a single alert instance.
+type AlertInstance struct {
+	OrgID             int64 `xorm:"org_id"`
+	AlertDefinitionID int64 `xorm:"alert_definition_id"`
+	Labels            InstanceLabels
+	LabelsHash        string
+	CurrentState      InstanceStateType
+	CurrentStateSince time.Time
+	LastEvalTime      time.Time
```

Comment:
If we use this field for date comparison, maybe it's preferable [to be saved as epoch](https://github.com/grafana/grafana/tree/master/pkg#date-comparison) in the database